tests/network/bond/test_bond_modes.py

```diff
@@ -7,6 +7,7 @@
 from contextlib import contextmanager
 
 import pytest
+from ocp_resources.resource import Resource
```

Comment:
Is this needed? `VirtualMachineForTests` inherits from `Resource` and has inner classes you use

Reply:
It is not clear what is better, this is the outcome of bad use of inheritance from my perspective.

If we use it from the object itself, it does not expose the source of it.
On the other hand, if we use that, we win one less line here.

I will move the waiting to the condition to the libs/vm module and then it will not be here.
Can we do it in a follow up?

Reply:
Just from readability perspective
`VirtualMachineForTests.Condition.Type.AGENT_CONNECTED` makes more sense then `Resource.Condition.Type.AGENT_CONNECTED` (since this condition is relevant only to VM)

Ack for follow up pr

Reply:
> Just from readability perspective `VirtualMachineForTests.Condition.Type.AGENT_CONNECTED` makes more sense then `Resource.Condition.Type.AGENT_CONNECTED` (since this condition is relevant only to VM)

From readability I agree.
The problem is that it lies, as the definition is done on the top level `Resource`.
I think this is a good middle-ground, I will define it on the VMI one which is more accurate.